cola/widgets/remote.py

```diff
@@ -3,6 +3,8 @@
 import time
 import os
 
+from cola.models import prefs
+
 from cola import resources
```

Comment:
I must've not noticed when these absolute imports snuck in.

We should use `from ..models import prefs` and `from . import resources` instead of mentioning the `cola` package here.

I'll go ahead and update the pre-existing import on the main branch, so you'll have to rebase to pickup that tweak shortly.

Reply:
These imports should come later in the internal section, not up here in the stdlib and third-party section.